src/strSlice.ml

```diff
@@ -142,3 +170,10 @@ let fold_left f init s =
 (*   let s = of_string "abcde" in *)
 (*   assert (fold_left (fun _ n -> n + 1) 0 s = 5); *)
 (*   assert (fold_left (fun c s -> String.make 2 c ^ s) "" s = "eeddccbbaa") *)
+
+let trim s =
+  let is_whitespace = function
+    | ' ' | '\t' | '\010' .. '\013' -> true
+    | _ -> false
+  in
+  drop_while is_whitespace (drop_last_while is_whitespace s)
```

Comment:
Not necessary for this pr but: tho it's only a few lines of code, but I still wonder if it's worth trying to share the function defined at https://github.com/ocaml/omd/blob/7b59e5926a1f811e1d89b4b37b163a459692d308/src/parser.ml#L11-L13

Could mean adding a util module or....